docs/source/gs-processing/usage/amazon-sagemaker.rst

```diff
@@ -62,21 +62,30 @@ job, followed by the re-partitioning job, both on SageMaker:
         --instance-type ${INSTANCE_TYPE} \
         --job-name "${GRAPH_NAME}-${INSTANCE_COUNT}x-${INSTANCE_TYPE//./-}-${NUM_FILES}files" \
         --num-output-files ${NUM_FILES} \
+        --do-repartition True \
         --wait-for-job
 
-    # This will run the follow-up re-partitioning job
+Launch the gs-repartition job on Amazon SageMaker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+In the above we have set `--do-repartition True` to perform the re-partition step on the Spark
+leader instance. The re-partitioning job runs on a single instance, so for large graphs you will
```

Comment:
```suggestion
leader instance, when the re-partitioning job runs on a single instance. For large graphs you will
```

Reply:
>In the above we have set `--do-repartition True` to perform the re-partition step on the Spark
leader instance, when the re-partitioning job runs on a single instance. For large graphs you will [..]

By `when the re-partitioning job runs on a single instance` do we mean to say when it's possible to run re-partitioning on a single instance?